# Anchor message structure. qf is the quality of this measurement and dist_qf the quality of the estimated distance.

Header header

float64 x ## in m
float64 y ## in m
float64 z ## in m
float64 qf ## in percent
float64 distance ## in m
float64 dist_qf ## in percent
